pkg/uki/install.go

```diff
@@ -159,6 +160,9 @@ func (i *InstallAction) Run() (err error) {
 		return fmt.Errorf("removing artifact set with role %s: %w", UnassignedArtifactRole, err)
 	}
 
+	// SelectBootEntry sets the default boot entry to the selected entry
+	err = action.SelectBootEntry(i.cfg, "active")
```

Comment:
also error handling afterwards, no? or at least logging if it's not meant to break

Reply:
good point, done